modules/extensions/scripts/install_OPA_gatekeeper.template.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2017, 2021 Oracle Corporation and/or affiliates.
```

Comment:
Was this file created in 2017? If not, you should only list 2021.

Reply:
Changed to 2021